MergerService/src/run.cs

```diff
@@ -204,6 +212,11 @@ public void Start()
                     }
                     finally
```

Comment:
Does `finally` execute after `continue` statement? I couldn't find an answer to this, we should test to make sure.

Reply:
finally does not execute after continue, i changed the code and take the continue statement outside from the catch.
check if the logic is ok.

Reply:
From what I can see this looks good, please be sure to check that it works in all cases.

Reply:
checked